docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -0,0 +1,231 @@
+= 5.3.0 Release Notes
+
+== New Features
+
+* **xref:integrate:kafka-connect-connectors.adoc[Connector for Kafka Connect source (BETA)]**: Import data from an external system directly into a Hazelcast data pipeline
+without the need of a Kafka cluster.
+* **xref:integrate:mongodb-connector[Connector for MongoDB (BETA)]**: Read and write from/to MongoDB via this connector and execute SQL queries on Mongo collections directly from Hazelcast.
+* **https://github.com/hazelcast/hazelcast/blob/master/docs/design/sql/16-partition-aware-client.md[Partition-Aware SQL Client]**: Send the SQL commands only to the members having the relevant data, which reduces the network hops and improves the query performances.
+
+== Breaking Changes
+
+* Renamed the `DataLinkFactory` interface as `DataConnection`.
```

Comment:
if think there might have been some breaking changes in the PRs I created or merged, but I do not remember now in which ones

Reply:
https://github.com/hazelcast/hazelcast/pull/24282
https://github.com/hazelcast/hazelcast/pull/23848